legal-api/src/legal_api/models/business.py

```diff
@@ -524,14 +526,16 @@ def json(self, slim=False):
             'associationType': self.association_type,
             'allowedActions': self.allowable_actions
         }
+
+        if flags.is_on('enable-legal-name-fix'):
```

Comment:
Moved this back to here in the big JSON. The only use I believe to have it in slim JSON is fore the search endpoint.

Reply:
It is needed in the slim json for business search + person search updates (this was the original use case we added the slim json for as the call was timing out for several businesses)

Reply:
also I don't think it needs to check the flag anymore as you have released legal name fixes?

Reply:
Got it, will revert this change back Kial.

Also, yeah I agree. We can probably get rid of the FF.

Reply:
awesome thank you 👍

Reply:
@kialj876 Kial, from my understanding, you only use on your side the search endpoint as well right? If that's the case, if the alternateNames is only there for firms (SPs and GPs) in slim JSON, does that cause issues on your side?

Reply:
for the flows I am talking about here we use the business endpoint NOT the search endpoint